Gemfile

```diff
@@ -153,6 +153,10 @@ gem 'remotipart', '~> 1.4.4' # Allows file upload in AJAX forms
 
 gem 'rails-static-router'
 
+gem 'caxlsx', '>= 3.0' # Write content to an xlsx file
+gem 'caxlsx_rails', '~> 0.6.2'
+
+# to avoid warnings after rails 6.1.7.2 update - see https://github.com/ruby/net-imap/issues/16
```

Comment:
this comment is no longer needed, not sure why it has reappeared but was fixed with ruby 3.1, but net-ftp is still needed